doc/wg/core/notes/core-notes-2020-10-23.md

```diff
@@ -0,0 +1,491 @@
+  detail. Thinking about the interface raises questions such as
+
+  - what would this mechanism mean for the userspace APIs / how would
+    it change them
+  - how would the kernel check that a process is allowed to do what it
+    tries to do
+
+
+* Amit: This conversation might be getting away from many. Maybe it
+  would be good to continue and summarize the discussion on a future
+  call?
+
+* Brad: What came up in the OpenTitan call but not here is the
+  distinction between a _global_ identifier vs. a _local_ identifier,
+  where _global_ would be an identifier that is namespaced under an
+  organisation and would not collide, whereas a locally unique
+  identifier would be unique across a board.
+
+  My proposal is to add a TLV, not claiming it solves every issue, and
+  is inserted by tockloader which ensures it is locally unique. When
+  it is updated, tockloader ensures that it does not change its
+  identifier.
+
+* Leon: What should be then to about the app id already in the process
+  struct? Rename it to be a process identifier instead?
+
+  Also, did we now come to a conclusion about the locally duplicate
+  ids? In Alistair's current proposal, those would still exist.
+
+* Brad: First question is just variable naming.
+
+  Second question: the identifier would be unique across a board,
+  which the loading tool (tockloader) would enforce.
+
+* Phil: Why does it need to be unique?
+
+* Brad: For our scheduling case it must be unique to be useful.
+
+  Our use case is scheduling over multiple power cycles. Every process
+  must need to be identified persistent across reboots.
+
+* Leon: Could you just use the position of the process in the process
+  array? That is already contained in `AppId` and would remain
+  constant, assuming that the order of apps does not change.
+
+* Brad: It would be possible to implement it this way, but we prefer
+  not to rely on that.
+
+* Phil: It's tricky to introduce an id which we are trying to use for
+  many things simultaneously, where the actual requirements might be
+  different.
+
+  A simple proposal to go with the TLV that Alistair proposed,
+  separating the TLV for the app identifier and the TLV for granting
+  access to capsules. For now, apps only have a single storage access
+  TLV. In the future it is possible that an app could have multiple
+  storage TLVs.
+
+* Brad: The notion of a single TLV with dynamic size already exists,
+  so this could be used.
+
+* Hudson: Calling out one of Jonathan's quote from the discussion in
+  May: "I'm concerned that developing an application specifically for
+  persistent storage will result in an application id design that is
+  poorly suited for the other use cases, that would either hurt those
+  use cases or result in us having multiple types of application id
+  which would be confusing, which hurts security and would waste flash
+  space."
+
+* Jonathan: I'm happy to reiterate on my proposal. I see a few
```

Comment:
Line 461: "reiterate" should have been "iterate" (sorry if I didn't speak clearly).

Reply:
Thanks for the hint. Entirely possible I just wrote down the wrong word. Addressed it in a fixup commit.